ui-v2/app/components/consul-service-list/index.hbs

```diff
@@ -1,24 +1,25 @@
 {{#if (gt items.length 0)}}
 <ListCollection @items={{items}} class="consul-service-list" as |item index|>
   <BlockSlot @name="header">
-    <dl class={{service/health-checks item}}>
-      <dt>
-        Health
-      </dt>
+{{#let (get proxies item.Name) as |proxy|}}
+  {{#let (service/health-checks item proxy) as |health|}}
+    <dl class={{health}}>
       <dd>
```

Comment:
Not sure if you meant to remove this ^. Completely up to you if you want to leave it in or not.